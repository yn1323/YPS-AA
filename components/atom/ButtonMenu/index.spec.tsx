import { Delete } from '@material-ui/icons'
import React from 'react'
import renderer from 'react-test-renderer'
import Component from '.'

test(Component.name, () => {
  const component = renderer.create(
    <Component icon={<Delete />} clickHandler={() => console.log('OK')}>
      LABEL
    </Component>
  )
  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()

  const component2 = renderer.create(
    <Component
      icon={<Delete />}
      clickHandler={() => console.log('OK')}
      showArrowDown={false}
    >
      LABEL
    </Component>
  )
  const tree2 = component2.toJSON()

  expect(tree2).toMatchSnapshot()
})
