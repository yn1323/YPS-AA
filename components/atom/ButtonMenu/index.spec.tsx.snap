// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonMenu 1`] = `
<button
  className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-colorInherit"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="MuiButton-label"
  >
    <span
      className="MuiButton-startIcon MuiButton-iconSizeMedium"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"
        />
      </svg>
    </span>
    LABEL
    <span
      className="MuiButton-endIcon MuiButton-iconSizeMedium"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M7.41 8.59L12 13.17l4.59-4.58L18 10l-6 6-6-6 1.41-1.41z"
        />
      </svg>
    </span>
  </span>
</button>
`;

exports[`ButtonMenu 2`] = `
<button
  className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-colorInherit"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="MuiButton-label"
  >
    <span
      className="MuiButton-startIcon MuiButton-iconSizeMedium"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"
        />
      </svg>
    </span>
    LABEL
  </span>
  <span
    className="MuiTouchRipple-root"
  />
</button>
`;
